Sources/StrongContractResponse/All/URLQueuryItem.swift

```diff
@@ -4,14 +4,16 @@
 //
 //  Created by Scott Lydon on 4/6/24.
 //
-
 import Foundation
 
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+
 /// Needs an extension for Codable.
 extension URLQueryItem: Codable {
-    enum CodingKeys: String, CodingKey {
-        case name
-        case value
+    private enum CodingKeys: String, CodingKey {
+        case name, value
     }
```

Comment:
Linters tend to prefer enumerate to have their own line per case.  I don't mind though if there are only two cases